sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -26,7 +26,78 @@ This product is fully supported and enabled by default as of {product-title} 4.1
++
+Following SELinux policy on the host guarantees full isolation of the host file system from the sandboxed workload by default. This also provides stronger protection against potential security flaws in the `virtiofsd` daemon or QEMU.
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to `container_file_t`, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
++
+Automatic relabeling is not an option when mounted files or directories are expected to have specific SELinux labels on the host. Instead, you can set custom SELinux rules on the host to allow the `virtiofsd` daemon to access these specific labels. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+
+* Some {sandboxed-containers-operator} pods that use container CPU resource limits to increase the number of available CPUs for the pod. These pods might receive fewer CPUs than requested. If the functionality is available inside the container, you can diagnose CPU resource issues by using `oc rsh <pod>` and running the `lscpu` command:
++
+[source,terminal]
+----
+$ lscpu
+----
++
+.Example output
++
+[source,terminal]
+----
+CPU(s):                          16
+On-line CPU(s) list:             0-12,14,15
+Off-line CPU(s) list:            13
+----
++
+The list of offline CPUs will likely change unpredictably from run to run.
++
+As a workaround, you can use a pod annotation to request additional CPUs rather than setting a CPU limit. CPU requests that use pod annotation are not affected by this issue, because the processor allocation method is different. Rather than setting a CPU limit, the following `annotation` must be added to the metadata of the pod:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    io.katacontainers.config.hypervisor.default_vcpus: "16"
+----
++
+(link:https://issues.redhat.com/browse/KATA-1376[*KATA-1376*])
+
+* The progress of the runtime installation is shown in the `status` section of the `kataConfig` custom resource (CR). However, the progress is not shown if all of the following conditions are true:
+
+** There are no worker nodes defined. You can run `oc get machineconfigpool` to check the number of worker nodes in the machine config pool.
+** No `kataConfigPoolSelector` is specified to select nodes for installation.
+
++
+In this case, the installation starts on the master nodes because the Operator assumes it is a converged cluster where nodes have both master and worker roles. The `status` section of the `kataConfig` CR is not updated during the installation. (link:https://issues.redhat.com/browse/KATA-1017[*KATA-1017*])
```

Comment:
You could change "master nodes" to "control plane nodes", to follow the conscious language guidelines. For more information, see [Using conscious language](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language) in the OpenShift documentation guidelines.

This applies to all references to the word "master" in this PR, except for any direct code references.

Reply:
I'd rather not do that (and same for workers above) - master and worker nodes are the Kubernetes technical terminology whether one likes it or not and renaming things will confuse readers.  I myself as a reader would probably be able to relate "control plane nodes" to master nodes but I'd be confused why they don't call the masters the masters (is control plane nodes a superset of master nodes? or a subset? or what is the relation between the two).

Reading "compute nodes", it wouldn't occur to me that worker nodes are meant.

Reply:
I'd like to give the entire kata team and the PM a heads up before I change all references to master/worker in the docs. I'll create a later PR to review all these references and make those changes.

Reply:
Sounds good, makes sense.

Reply:
Thank you both for your responses.

I have subsequently discussed this point with @bergerhoffer and we both agree that we should avoid introducing new instances of the word "master" when possible'. @kalexand-rh can you please advise as to how we should proceed? Thanks!

Reply:
This known issue has existed in previous versions, so we aren't introducing a new use of "master". Half of these known issues were copied over from last release.

Reply:
This looks like you're talking about the OCP nodes, so they should be called "[control plane](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#control-plane)."

If you're talking about references in _code_ that have not yet been updated, "master" is ok to use, but _only_ in that context. If you're genuinely concerned about your users being confused about "control plane" versus "master," you can follow this [previous guidance](https://github.com/openshift/openshift-docs/commit/6a0622b49885e7d6aa657c96a767e5c7a4a5c33f) about pairing the two. (It's been deprecated because we stopped allowing new instances of "master" several releases ago.)

Reply:
Changed both instances of "master" to "control plane".